### PR TITLE
Modified requested method for reconfigure disk_remove.

### PR DESCRIPTION
**Pull request by billfitzgerald0120**
Updated spec

fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1644351

### CODE REVIEW

**Comment by billfitzgerald0120** [comment]
@miq-bot add_reviewer @tinaafitz
@miq-bot add_label bug

**Comment by billfitzgerald0120** [comment]
This line is really a problem:   disk_num = disk['disk_name'].match(/_(\d).vmdk/)"

File: content/automate/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/requested.rb [hunk 1]
```diff
@@ -146,6 +146,12 @@ def vmdb_object(model, id)
   $evm.vmdb(model, id.to_i) if model && id
 end
 
+def get_disk_size(disk_name)
+  mydisk = $evm.vmdb('disk').find_by(:filename => disk_name)
+  ManageIQ::Automate::System::CommonMethods::Utils::LogObject.log_and_raise("ERROR - Disk_name not found for Reconfiguration", @handle) if mydisk.nil?
```
Comment (mkanoor) [inline content/automate/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/requested.rb]: @billfitzgerald0120 
Do we need a spec for this?
Also we are missing the method instance yaml with the list of embedded methods

**Comment by tinaafitz** [comment]
@miq-bot add_label hammer/yes, gaprindashvili/yes
@simaishi yes, thanks for the reminder.

**Comment by simaishi** [comment]
Hammer backport details:

```
$ git log -1
commit cd49be5e3cb81fcc2415fe59bc7d9f138fe96413
Author: Madhu Kanoor <mkanoor@redhat.com>
Date:   Wed Nov 14 14:29:12 2018 -0500

    Merge pull request #461 from billfitzgerald0120/quota_vm_reconfig_disk_remove
    
    Modified requested method for reconfigure disk_remove.
    
    (cherry picked from commit c88a344807a704f4e3f1c0c0a53a80245f7ee08b)
    
    Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1644351
```

**Pull request merged**